test/kpod_ps.bats

```diff
@@ -0,0 +1,281 @@
+#!/usr/bin/env bats
+
+load helpers
+
+IMAGE="redis:alpine"
+ROOT="$TESTDIR/crio"
+RUNROOT="$TESTDIR/crio-run"
+KPOD_OPTIONS="--root $ROOT --runroot $RUNROOT ${STORAGE_OPTS}"
+
+@test "kpod ps default" {
```

Comment:
@umohnani8 No containers present or running - just a test that no containers = no output